server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java

```diff
@@ -282,6 +282,7 @@ public ClusterMetadataManifest writeIncrementalMetadata(
         if (updateGlobalMetadata || previousManifest.getGlobalMetadataFileName() == null) {
             globalMetadataFile = writeGlobalMetadata(clusterState);
         } else {
+            logger.debug("Global metadata has not updated in cluster state, skipping upload of it");
```

Comment:
can we add a debug log also for IndexMetadata if new files are being uploaded?

Reply:
We had the log for index metadata as well at trace level. Made it to debug log to maintain consistency. 

https://github.com/opensearch-project/OpenSearch/blob/b5299f13e0ca9a5f6979e8cb50137682e777b095/server/src/main/java/org/opensearch/gateway/remote/RemoteClusterStateService.java#L308-L313